storage/filestore/store_test.go

```diff
@@ -213,9 +226,301 @@ func TestDeleteWhileReading(t *testing.T) {
 		if info.IsDir() {
 			return nil
 		}
-		return errors.New("found file " + path)
+		return errs.New("found file %q", path)
 	})
 	if err != nil {
 		t.Fatal(err)
 	}
 }
+
+func writeABlob(ctx context.Context, t testing.TB, store *filestore.Store, blobRef storage.BlobRef, data []byte, formatVersion storage.FormatVersion) {
```

Comment:
Here a store testing context would be helpful to simplify the code, so you don't have to add ctx/t parameter everywhere.

```
type StoreContext struct {
    ctx context.Context
    t     testing.TB
    store *filestore.Store
}

func (ctx *StoreContext) WriteBlob(blobRef storage.BlobRef, data []byte, formatVersion storage.FormatVersion) {
...
```

Reply:
Noted, but it seems fine for what it needs to do right now.